src/styles/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
```

Comment:
Move box-sizing to * selector

Reply:
what means '* selector', could you explain please?